examples/mobile/HomeApp/js/app.js

```diff
@@ -125,6 +126,24 @@ import Storage from "./clipping-storage.js";
 				(app2.isActive) ? 0 : -1 : 1
 		});
 
+		updateManifests();
+
+		appsList = appsList.forEach((app) => {
+			if (app.isActive) {
```

Comment:
only sorting for active apps?
we should sort
"Control cards are always before service cards" regardless if card is active or not.